samwise.js

```diff
@@ -2,16 +2,20 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.